website/pages/docs/nia/installation/run.mdx

```diff
@@ -10,14 +10,21 @@ description: >-
 
 1. Move the `consul-terraform-sync` binary to a location available on your `PATH`.
 
-```
-$ mv ~/Downloads/consul-terraform-sync /usr/local/bin/consul-terraform-sync
-```
+  ```shell-session
+  $ mv ~/Downloads/consul-terraform-sync /usr/local/bin/consul-terraform-sync
+  ```
 
-2. Create the config.hcl file, all the options are available [here](https://github.com/hashicorp/consul-terraform-sync/blob/master/docs/config.md#consul-terraform-sync).
+2. Create the config.hcl file, all the options are available [here](/docs/nia/installation/configuration).
 
 3. Run consul-terraform-sync.
 
-```
-./consul-terraform-sync -config-file <config.hcl>
-```
+  ```shell-session
+  $ ./consul-terraform-sync -config-file <config.hcl>
```

Comment:
Do you mind including the removal of the `./` path prefix with your changes? If the binary is moved in to bin/path from step 1, I don't think the `./` local prefix works verbatim following the steps.

```suggestion
  $ consul-terraform-sync -config-file <config.hcl>
```

Reply:
Good point! thanks for catching